docs/azure-config.md

```diff
@@ -50,7 +50,8 @@ az storage account create \
     --kind BlobStorage \
     --access-tier Hot
 
-# Create the blob container named "ark". Feel free to use a different name; you'll need to
+# Create the blob container named "ark". Feel free to use a different name, preferrably unique to a 
+# single Kubernetes cluster. See the FAQ for more details. You'll need to
```

Comment:
could you add a link to the FAQ here like on the other pages?

Reply:
I split the section up so Markdown would render the FAQ link, not sure if that's desirable on your end though.